routes/messages/UI/ButtonGenerator.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Dynamically generates messenger buttons
+ * Params:
+ * - content: (Array) containing Button object values
+ */
+module.exports = async (content) => {
+  const { buttons } = content;
+  return generateButtons(buttons);
+};
+
+const generateButtons = function(btnArr) {
+  return btnArr.map((e) => {
+    return {
+      type: e.type,
+      title: e.title,
+      url: null || e.url,
+      payload: null || e.payload
```

Comment:
null || e.payload will cause it to always default to e.payload (which is fine, undefined works just as well).  If you want it to default to null, you'd just have to reverse the order! (`url: e.url || null`)